code/game/objects/items/devices/powersink.dm

```diff
@@ -99,6 +99,7 @@
 				"<span class='italics'>You hear a click.</span>")
 			message_admins("Power sink activated by [ADMIN_LOOKUPFLW(user)] at [ADMIN_VERBOSEJMP(src)]")
 			log_game("Power sink activated by [key_name(user)] at [AREACOORD(src)]")
+			notify_ghosts("[user] has activated a [name]!", title = "Something's Interesting!", source = src, flashwindow = FALSE, action = NOTIFY_FOLLOW)
```

Comment:
This title kinda sucks, it feels very "default". Could you change it to something more appropriate?

Reply:
We're being sucked!
Ugly [name] has activated powersink!

Reply:
I've changed it to `"Someone's plan is going into effect! (Click to follow)"` however it could probably do with some more suggestions, im kinda at a loss on it should be

Reply:
changed again to `An electrifying occurrence! (Click to follow)`

Reply:
Still not very clear to people what is happening. Just something like the message for the title would be good too. There's no need to be overly fancy about it

Reply:
i think its fine, the title only appears when you're hovering over the icon, and the description explains whats going on. 
The description is also what gets put in chat

Reply:
"The station's powernet is being sapped!" could work